src/App.js

```diff
@@ -34,7 +35,11 @@ const configValues = {
 setConfig(configValues);
 
 function App() {
-  return <div className="bc_services-list-app"></div>;
+  return (
+    <div className="bc_services-list-app">
```

Comment:
can we make this more generic, i don't have a good suggestion. Maybe just `dg_app`?

Reply:
maybe bc_app

Reply:
looks like we dont even use this css so its been removed